### PR TITLE
new markers test pls

### PR DESCRIPTION
**Pull request by Almightywesley**
what do you think of it?

### CODE REVIEW

**Review by Hicham010 — APPROVED** [review]
Nice work on the markers!

File: app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MapFragment.java [hunk 1]
```diff
@@ -1,166 +1,161 @@
...
-import nl.hogeschoolrotterdam.projectb.util.SimpleAnimatorListener;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
 /**
  * Created by maartendegoede on 20/03/2019.
- * Copyright © 2019 Anass El Mahdaoui, Hicham El Marzgioui, Wesley de Man, Maarten de Goede all rights reserved.
+ * Copyright © 2019 Anass El Mahdaoui, Hicham El Marzgioui, Michaël van Asperen, Wesley de Man, Maarten de Goede all rights reserved.
```
Comment (MrTheGood) [inline app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MapFragment.java:40]: Why did you add this?

File: app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MapFragment.java [hunk 2]
```diff
@@ -1,166 +1,161 @@
...
+
+
+        }catch (IOException e){
+        }
+        if (list.size() > 0){
+            Address address = list.get(0);
+            //Toast.makeText(this, address.toString(), Toast.LENGTH_SHORT).show();
+            CameraPosition search = CameraPosition.builder().target(new LatLng(address.getLatitude(),
+                    address.getLongitude())).zoom(10).bearing(0).tilt(0).build();
+            mGoogleMap.moveCamera(CameraUpdateFactory.newCameraPosition(search));
```
Comment (MrTheGood) [inline app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MapFragment.java:95]: And why did almost this entire file get deleted, including the tooltip code?

**Review by MrTheGood — CHANGES_REQUESTED** [review]
It seems like you reverted a lot of code and broke everything. I cannot accept this.

**Comment by MrTheGood** [comment]
Everything is pushed to a different branch now. I'm closing this.
